### PR TITLE
Pop up

### PR DESCRIPTION
**Pull request by Luffytaro22**
### Implemented a Pop-Up Projects
* Responsive.
* Added a new JS file with the information on the project's cards.
* Eliminated the project's elements in HTML.
* Pop-up's for every project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -65,67 +65,10 @@ <h3>Multi-Post Stories</h3>
                 </div>
             </div>
             <div class="works-container" id="works-container">
```
Comment (V-Blaze) [inline index.html:70]: - [ ] You did great implementing the details pop. :+1: :100: Your pop-up works great but I think there are some styling issues you need to improve on before moving to the next milestone. 
- Looking at the Fimga design, when the pop-up is open, the background is blurred, which helps keep the pop-up window in focus, As part of the requirement for this milestone is to replicate the given design as much as you can, I'll suggest you take a look at this and add the blur background styling. :waffle: :smile: 
- Also following the given design, the Project image, takes more width than the content, I'll suggest you adjust their width to accurately follow the given design. :heavy_check_mark: :smile: 

- - Your Current design:
Selection_041

- - Figma design:
Selection_040

File: index.html [hunk 2]
```diff
@@ -65,67 +65,10 @@ <h3>Multi-Post Stories</h3>
...
-                        <li>HTML</li>
-                        <li>CSS</li>
-                        <li>Javascript</li>
-                    </ul>
-                    <a href="https://luffytaro22.github.io/Calculator/" target="_blank" rel="noopener" class="hide"><button type="button">See project</button></a>
-                </div>
             </div>
+            <div id="pop-up">
+            </div>
+            <script src="projects.js"></script> <!-- The projects cards -->
```
Comment (V-Blaze) [inline index.html]: - [x] Following the best practices, You don't add Javascript files at the center of your code, You can add this at the bottom of this file or at the top using the defer keyword. :+1: :100:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Luffytaro22  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Pull request summary is descriptive✔
-  Meaningful commit messages👋 
-  Readme look professional✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @Luffytaro22 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] Great looking UI design :+1: 
- [ ] Descriptive PR `title` and `summary` :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**